src/ProjectOrigin.WalletSystem.IntegrationTests/Repositories/CertificateRepositoryTests.cs

```diff
@@ -123,6 +123,32 @@ public async Task GetAllOwnedCertificates()
         );
     }
 
+    [Theory]
+    [InlineData(SliceState.Registering)]
+    [InlineData(SliceState.Sliced)]
+    [InlineData(SliceState.Slicing)]
+    [InlineData(SliceState.Transferred)]
```

Comment:
Could add available as well and either the count or boolean flag if any result should be found, then all scenarios is tested

Reply:
@duizer  but is up to you

Reply:
I like it. I have it changed it.